homeassistant/core.py

```diff
@@ -373,6 +374,35 @@ def from_dict(cls, json_dict):
         return cls(json_dict['entity_id'], json_dict['state'],
                    json_dict.get('attributes'), last_changed, last_updated)
 
+    def as_number(self):
```

Comment:
Yep

On Thu, Feb 11, 2016 at 4:25 PM, Dan Smith <notifications@github.com> wrote:

> In homeassistant/core.py
> <https://github.com/balloob/home-assistant/pull/1209#discussion_r52690498>
> :
>
> > @@ -373,6 +374,35 @@ def from_dict(cls, json_dict):
> >          return cls(json_dict['entity_id'], json_dict['state'],
> >                     json_dict.get('attributes'), last_changed, last_updated)
> >
> > +    def as_number(self):
>
> Sure, so usage would be:
>
>   helpers.state.state_as_number(state)
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant/pull/1209/files#r52690498>.
>



-- 
PaulusSchoutsen.nl
It's nice to be important but it's more important to be nice.